.github/workflows/test-providers.yml

```diff
@@ -12,8 +12,7 @@ jobs:
         with:
           node-version: "14"
       - run: rm -rf build
-      - run: yarn install
-      - run: yarn bootstrap
-      - run: yarn build
+      - run: pnpm install
```

Comment:
Missing the actions/cache here :)

Reply:
I thought that only an update from lerna cache was required. Will update the cache on test providers as well 👍